3p/ampcontext.js

```diff
@@ -312,8 +313,22 @@ export class AbstractAmpContext {
       this.setupMetadata_(this.win_.name);
     }
   }
-}
 
+  /**
+   * Send 3p error to parent iframe
+   * @private
+   */
+  report3pError_() {
+    this.win_.onerror = message => {
+      if (message) {
+        this.client_.sendMessage(MessageType.USER_ERROR_IN_IRAME, dict({
+          'message': message,
+        }));
+      }
+      return false;
```

Comment:
just curious, do we have to return `false` here?

Reply:
if we don't return false then it would be undefined, which is false. But i think we should explicitly return false to make it readable